storage/sql.go

```diff
@@ -324,6 +324,12 @@ func (s *SQL) DeleteStaleProjects(stale time.Duration) error {
 	return nil
 }
 
+func (s *SQL) TotalProjectCount(count *int64) error {
+	return s.db.Model(&model.Project{}).
+		Count(count).
+		Error
+}
```

Comment:
minor: Not a big fan of this method signature... it seems a bit weird.
could we just do: `func (s *SQL) TotalProjectCount() (int64,error)`

Reply:
I feel the whole storage interface should be refactored to not return values by argument pointers but by returning the values. This is not an idiomatic way in Go and the design decision was made a while ago and I feel we should correct it at this point. Also we should consider adding context as the first argument.